packages/signing-utils/src/mocha-hooks.ts

```diff
@@ -0,0 +1,11 @@
+import * as os from 'os';
+
+export const mochaHooks = {
+  beforeEach() {
+    // @ts-expect-error Stupid mocha binding properties to `this`
+    const test = this.currentTest ?? this.test;
+    if (os.platform().toLowerCase().includes('win32')) {
+      test.skip();
+    }
```

Comment:
I was pretty sure that just `this.skip` is a thing in beforeEach and you don't need to specifically get the spec first, just wondering if this expect-error is needed 🤔

Reply:
Because the hooks are declared in a regular object, typescript is unable to infer that a skip method exists on this object. This is still true, even if I explicitly type beforeEach.

I did make your suggested change though, we just still need the expect error.

Reply:
Ohhh, right, because you're not in an actual test file calling a global beforeEach, totally missed that, makes sense!